### DISCUSSION TITLE
Collaborating with Martin Tile Server

### DISCUSSION THREAD

**Discussion by nyurik** [jessekrubin/utiles#14] [Ideas]
Hi, thanks for starting this project!  I am maintaining [Martin tile server](https://maplibre.org/martin/) (part of MapLibre org, written in Rust) that has a lot of related technology, and I would love to consolidate some of the fundamental tile utilities in one spot. I already have a lot of MBTiles related functionality as well (i saw you had thoughts about that in the README), plus tile generation and other features.  What are you thoughts on joining our efforts?  We can also sync up online or over video (most of MapLibre discussion is on OSM-US [slack](https://slack.openstreetmap.us/)) Thx!

CC: @jessekrubin  @dan-costello

**Comment by jessekrubin** [comment]
@nyurik Howdy! I have been following martin quite closely, AND I have been meaning to contact you (mr sqlite-hashes/pmtiles/stadiamaps/martin)!

Would love to collab! How would you like to combo our efforts?

I am very open to just about any suggestions/changes you want to make!

I was thinking of moving the python-lib out of the repo-root.

I also have many things I would like to do w/ utiles, and about 50+ experiments I would like to investigate.

Where do you wanna start? lmk if you want my brain dump.
